src/modules/customer/containers/HappyUserPage/components/WizardForm/constants.js

```diff
@@ -0,0 +1,45 @@
+export const FORM_NAME = 'happyUserWizardForm';
+
+export const DESTROY_ON_MOUNT = false;
+export const FORCE_UNREGISTER_ON_UNMOUNT = true;
+
+export const formNamesFirstName = 'firstName';
```

Comment:
i think these names will be reused all over the app, so move them to app/constants/wizardForm